pkgs/top-level/python-packages.nix

```diff
@@ -17407,6 +17410,8 @@ self: super: with self; {
     inherit (pkgs) zopfli;
   };
 
+  zotify = callPackage ../tools/misc/zotify { };
+
```

Comment:
Why is zotify here?

Reply:
I wasn't sure of the best place to put it, so I looked for where youtube-dl is and had it there. Where does it have to be instead?

Reply:
It should usually be in pkgs/top-level/all-packages.nix. I think python libraries are meant to go here.